.github/workflows/examples-efr32.yaml

```diff
@@ -146,6 +146,24 @@ jobs:
             /tmp/bloat_reports/
       - name: Clean out build output
         run: rm -rf ./out
+      - name: Build BRD4342A WiFi Soc variants
+        run: |
+          ./scripts/run_in_build_env.sh \
+             "./scripts/build/build_examples.py \
+                --enable-flashbundle \
+                --target efr32-brd4342a-light-skip-rps-generation \
+                --target efr32-brd4342a-lock-skip-rps-generation \
+                build \
+                --copy-artifacts-to out/artifacts \
+             "
+      - name: Prepare bloat report for brd4342a lock app
```

Comment:
I don't think the bloat check is here and for efr32 BRD2605a lock-app is relevant. it will produce the same result as the `bloat report for brd4338a lock app`

Reply:
Removed the bloat report for BRD4342A and BRD2605A boards and applied restyle issue.